### PR TITLE
Update11

### PR DESCRIPTION
**Pull request by zkline101**

### CODE REVIEW

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (zkline101) [inline .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata]: Looks like we did not include this file before which I don't think will hurt anything but after reading [this](https://stackoverflow.com/questions/65754188/should-i-add-swiftpm-to-gitignore) we may want to include it.

File: README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ That's it. ArcGIS Runtime SDK for iOS has been integrated into your project as a
 Try out some of the tutorials.
 
 ## Requirements
-- Xcode 12.0 or newer
+- Xcode 13.0 or newer
```
Comment (dg0yal) [inline README.md]: We don't need a min of Xcode 13 as far as i know. Any reason you know?
I have been using my Xcode 12.4 for all testing so far and it has worked.

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata (hunk 1 above)
Comment (dg0yal) [reply .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata]: Not so sure we should add this. All of our testing has been without it and things have worked so far. So why change. I read the post, and it didn't convince me. But perhaps get a second opinion for @philium

**Review by dg0yal — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (zkline101) [reply README.md]: Ah whoops sorry I was moving too quickly I was thinking iOS 13

**Comment by philium** [comment]
> Not so sure we should add this.

The problem is that currently, Xcode automatically creates this file whenever the package is opened, which is why we've checked in this file to other package repos we've created. However, the file is now [excluded](https://github.com/apple/swift-package-manager/blob/cfbe3609f872b411c2c9f7db8f85e490a357fdc7/Sources/Workspace/InitPackage.swift#L246) by default for packages created in Swift 5.4 and newer (see [PR](https://github.com/apple/swift-package-manager/pull/2674)). So I would say we should ignore the file.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+*.xcuserstate
+xcuserdata/
```
Comment (philium) [inline .gitignore]: Please use the [standard gitignore](https://github.com/apple/swift-package-manager/blob/41b97c87dfd37dc263abc4975a65b7d5042e00be/Sources/Workspace/InitPackage.swift#L257-L263) that SPM provides.

**Review by dg0yal — APPROVED** [review]

**Review by philium — APPROVED** [review]

**Pull request merged**